specification/authorization/resource-manager/Microsoft.Authorization/preview/2020-10-01-preview/examples/GetRoleAssignmentScheduleRequestByName.json

```diff
@@ -22,7 +22,7 @@
             "expiration": {
               "type": "AfterDuration",
               "endDateTime": null,
-              "duration": "08:00:00"
+              "duration": "PT8H"
```

Comment:
are these changes jsut fixing bugs in the swagger examples, or has the service contract changed?

Reply:
Yes this to just fix examples. The service contract is the same.